### PR TITLE
Enable launching the native Android posting experience

### PR DESCRIPTION
**Pull request by jayzes**

### CODE REVIEW

**Comment by ellodanger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a linked tracker story for this PR in the description</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">Nice, more deletions than insertions :red_circle:</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by jayzes** [comment]
@rynbyjn SGTM!

**Comment by rynbyjn** [comment]
@mkitt I think this is good to go now, but would like for you to maybe check out the comment button additions in [this commit](https://github.com/ello/webapp/pull/203/commits/17fec0eec1791b93eb4e8249c1d71504bcb3899a) 🤘 .

File: src/components/posts/PostRenderables.js [hunk 1]
```diff
@@ -252,6 +252,42 @@ export class PostBody extends PureComponent {
...
 
+const launchCommentEditorButtonStyle = css(
+  s.bgcBlack,
+  s.colorWhite,
+  s.sansRegular,
+  {
+    borderRadius: 5,
+    height: 50,
+    minWidth: '100%',
+    maxWidth: '640px',
```
Comment (mkitt) [inline src/components/posts/PostRenderables.js]: Can be just `640`

File: src/components/editor/Editor.css [hunk 2]
```diff
@@ -26,6 +26,10 @@
   transition: border-color var(--speed) ease;
 }
 
+.editor.LaunchCommentEditorButton.isComment::before {
+  border-right-color: #000;
+}
+
```
Comment (mkitt) [inline src/components/editor/Editor.css]: 👍 cool to just piggy back on what's already there.

File: src/components/posts/PostRenderables.js [hunk 3]
```diff
@@ -252,6 +252,42 @@ export class PostBody extends PureComponent {
...
+    paddingLeft: 20,
+    textAlign: 'left',
+  },
```
Comment (mkitt) [inline src/components/posts/PostRenderables.js]: You could use the shortcuts for[`s.pl20`](https://github.com/ello/webapp/blob/master/src/styles/jso.js#L135) for the `paddingLeft`  and [`s.leftAlign`](https://github.com/ello/webapp/blob/master/src/styles/jso.js#L268) for `textAlign` if you wanted. I'm not 100% sure these shortcuts are good or not. They make sense to me but might not be totally intuitive. Curious if you think we should hang onto them or maybe do away with them all together?

**Pull request merged**